main.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+
 echo "I want to be the first : Arin"
+
+echo "I modified this file."
```

Comment:
Are you sure you dont want to be the first?

Reply:
No, thank you.